saw-core/saw-core.el

```diff
@@ -0,0 +1,137 @@
+;;; saw-core.el --- A major mode for editing saw-core  -*- lexical-binding: t; -*-
```

Comment:
We should probably make a separate PR that adds `saw-core.el` to the master branch.

Reply:
Yeah, I think more people might use it. If nothing else, it's better than fundamental mode in emacs...

Reply:
See #157.